src/server/src/modules/student.ts

```diff
@@ -128,3 +127,52 @@ export const getJobApplications = async (studentId: number) => {
         ])
         .getMany();
 };
+
+export const searchStudents = async (
+    firstName: string,
```

Comment:
use IStudent['firstName'] type instead of string

Reply:
Done.